streams/jp.m3u

```diff
@@ -89,8 +89,8 @@ http://cdn.us195.jpnettv.live:1935/jptv/nhkbs1_540/playlist.m3u8
 https://nhkw-zh-hlscomp.akamaized.net/8thz5iufork8wjip/playlist.m3u8
 #EXTINF:-1 tvg-id="NHKChinese.jp",NHK Chinese Vision (720p)
 https://nhkw-zh-hlscomp.akamaized.net/ixxemlzk1vqvy44o/playlist.m3u8
-#EXTINF:-1 tvg-id="NHKEducationalTV.jp" status="error",NHK E Osaka
-http://cdn.us195.jpnettv.live:1935/jptv/kansi_nhk_edu_540/playlist.m3u8
+#EXTINF:-1 tvg-id="NHKEducationalTV.jp" status="online",NHK E
+https://cdn.us195.jpnettv.live/jptv/kanto_nhk_edu_540/chunklist_w470386795.m3u8
```

Comment:
```suggestion
https://cdn.us195.jpnettv.live/jptv/kanto_nhk_edu_540/playlist.m3u8
```

Reply:
you can take the version with ..nhk_edu_540/playlist.m3u8